### ISSUE TITLE
There is no burn event though this comment suggests otherwise

### ISSUE THREAD

**Issue by mswezey23** [OpenZeppelin/openzeppelin-solidity#1371]
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/f4d6f404424669e0eef7faa05fd89ab9945f265b/contracts/token/ERC20/ERC20Burnable.sol#L29

**Comment by frangio** [comment]
Good catch @mswezey23! The whole `_burn` override can actually be removed.

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/f4d6f404424669e0eef7faa05fd89ab9945f265b/contracts/token/ERC20/ERC20Burnable.sol#L28-L34

Are you interested in sending a pull request with these changes?

**Comment by mswezey23** [comment]
Next time I'll issue a PR for the changes. Looks like @Aniket-Engg beat me to it ;) 

I assume the same/similar occurred for the Mint event?
